governance.md

```diff
@@ -17,4 +17,16 @@ https://github.com/pangeo-data/governance
 
 ## Institutional Partners and Funding
 
+The Steering Council is the primary leadership for the project. No outside institution, individual or legal entity has the ability to own, control, usurp or influence the project other than by participating in the Project as Contributors and Council Members. However, because institutions are the primary funding mechanism for the project, it is important to formally acknowledge institutional participation in the project. These are Institutional Partners.
+
+An Institutional Contributor is any individual Project Contributor who contributes to the project as part of their official duties at an Institutional Partner. Likewise, an Institutional Council Member is any Project Steering Council Member who contributes to the project as part of their official duties at an Institutional Partner.
+
+With these definitions, an Institutional Partner is any recognized legal entity in the United States or elsewhere that employs at least one Institutional Contributor or Institutional Council Member. Institutional Partners can be for-profit or non-profit entities.
```

Comment:
`United States or elsewhere` is just everywhere, isn't it? What about `recognized legal entity in any country`

Reply:
Right!  I thought that was weird, too.